.travis.yml

```diff
@@ -19,10 +18,14 @@ matrix:
     - python: '3.6'
       env:
         - TOXENV=py36,report,coveralls
-    - python: 'pypy-5.4'
+    - python: '3.6'
```

Comment:
Why 3.6 twice (lines 18 and 21)?

Reply:
This is a matrix, so it is python 3.6 and env TOXENV=docs so it has a dedicated docs tasks

Reply:
Hmm... so using `- TOXENV=py36,report,coveralls,docs` generates a different output?
I'm trying to understand how it works.

Reply:
No, it will create one TOXENV=py32,report,coveralls and one with TOXENV=docs both running with travis python3.6 support